app/assets/stylesheets/components/_slider.scss

```diff
@@ -0,0 +1,199 @@
+$track-width: 16em;
+$track-height: .625em;
+$track-bg: $white;
+$track-fill: linear-gradient(90deg, $grey-3 25%, $grey-3 100%);
+
+$thumb-diameter: 1.5em;
+
+$option-colours: (
+  "leave-pot-untouched": $color-option-leave-pot-untouched,
+  "guaranteed-income": $color-option-guaranteed-income,
+  "adjustable-income": $color-option-adjustable-income,
+  "take-cash": $color-option-take-cash,
+  "whole-pot": $color-option-take-whole-pot,
+  "mix-options": $color-option-mix-options
+);
+
+
+// MIXINS
+@mixin track($flag: true) {
+  width: $track-width;
+  height: $track-height;
+  border-radius: $track-height / 2;
+  box-shadow: inset 2px 2px 0 darken($grey-2, 1.7);
+  border-right: 2px solid $grey-3;
+  border-bottom: 2px solid $grey-3;
+  background: $track-bg;
+
+  @if $flag {
+    background: $track-fill no-repeat $track-bg;
+    background-size: 0 100%;
+  }
+}
+
+@mixin thumb($option: "leave-pot-untouched") {
```

Comment:
Do we want to couple the slider component to being used in the context of an option?

If we passed in the colour instead of the map key we could remove the coupling.